react/features/filmstrip/reducer.js

```diff
@@ -145,46 +154,12 @@ ReducerRegistry.register(
                 visibleParticipantsEndIndex: action.endIndex,
                 visibleParticipants: state.remoteParticipants.slice(action.startIndex, action.endIndex + 1)
             };
-        case PARTICIPANT_JOINED: {
-            const { id, local } = action.participant;
-
-            if (!local) {
-                state.remoteParticipants = [ ...state.remoteParticipants, id ];
```

Comment:
@hristoterezov Is there a reason why you didn't add a check for fake participants here ? Shouldn't we avoid adding fake participants to the list of remote participants ?

Reply:
Every participant(fake or real) that is included in the array will have a thumbnails. We do render thumbnail for the fake participants (youtube videos). I don't see a reason why we should exclude them. Why do you think they must be included? Am I missing something?

Reply:
Oh I thought recorder was also considered as a fake participant, I see that only shared-video is considered a fake participant.